pippo-core/src/main/java/ro/pippo/core/ContentTypeEngines.java

```diff
@@ -93,25 +115,30 @@ public ContentTypeEngine registerContentTypeEngine(Class<? extends ContentTypeEn
     }
 
     /**
-     * Returns the first matching content type engine for a simple content type
-     * or a complex accept header like:
+     * Returns the first matching content type engine for a content type suffix (json, xml, yaml), a
+     * simple content type (application/json) or a complex accept header like:
      * <p/>
      * <pre>
      * text/html,application/xhtml+xml,application/xml;q=0.9,image/webp
      * </pre>
      *
-     * @param contentType
+     * @param contentTypeOrSuffix
      * @return null or the first matching content type engine
      */
-    public ContentTypeEngine getContentTypeEngine(String contentType) {
-        if (StringUtils.isNullOrEmpty(contentType)) {
+    public ContentTypeEngine getContentTypeEngine(String contentTypeOrSuffix) {
```

Comment:
From implementation I see a permutation: `suffixOrContentType` instead of `contentTypeOrSuffix`. Am I wrong?